lib/tpcal.py

```diff
@@ -170,5 +175,5 @@ def calibrate(points):
         Tpcal_point(-40, -40, "lower right-hand corner"),
     ], calibrate)
 
-# while True:
-#    pass
+while True:
```

Comment:
Why uncommented?
This endless while loop is usually not needed.

Reply:
I thought I needed it, but you are right. We can keep it commented